### PR TITLE
chore: add volunteer miraz-mondal

### PR DESCRIPTION
**Pull request by miraz-mondal**
I want to be a part of AskDevs as a volunteer as I love to help other person!

### CODE REVIEW

File: data/users/miraz-mondal.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Miraz Mondal",
+  "bio": "Web Developer",
+  "image": "https://ibb.co/mDsy4cf",
```
Comment (codewithvoid) [inline data/users/miraz-mondal.json:4]: Can you use your GitHub profile image here
https://avatars.githubusercontent.com/u/89645073?v=4

File: data/users/miraz-mondal.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "categories": [
+    {
+      "slug": "web-development",
+      "topics": ["Programming","Full Stack development","PHP","MERN"]
```
Comment (codewithvoid) [inline data/users/miraz-mondal.json:8]: mention programming in separate category with mention of programming language

```json
    {
      "slug": "programming",
      "topics": ["python", "javascript"]
    }
```

Comment (codewithvoid) [inline data/users/miraz-mondal.json:8]: better to mention `Full stack developer` in bio

**Comment by miraz-mondal** [comment]
@codewithvoid okay, making the changes ASAP

**Comment by miraz-mondal** [comment]
@codewithvoid please review it

**Comment by codewithvoid** [comment]
Perfect. Thanks @miraz-mondal for volunteering :)

**Pull request merged**